core-plugin/testSrc/com/google/gct/idea/debugger/CloudDebugProcessTest.java

```diff
@@ -72,7 +74,7 @@ public void testRegisterAdditionalActions_close() {
 
         assertEquals(3, leftToolbar.getChildrenCount());
         assertEquals(action0, leftToolbar.getChildActionsOrStubs()[0]);
-        assertEquals(action2, leftToolbar.getChildActionsOrStubs()[1]);
+        assertEquals(action3, leftToolbar.getChildActionsOrStubs()[1]);
```

Comment:
I think we're good if you just add a test for the parts of CloudDebugHelpAction that can be reasonably easily tested.